scripts/generate_dump

```diff
@@ -98,10 +190,10 @@ save_cmd() {
 #  None
 ###############################################################################
 save_cmd_all_ns() {
-    echo $1
-    echo $2
+    #echo $1
```

Comment:
You can remove the commented echo statements here.

Reply:
Done